src/specs/components/site-footer/demo/data-harmonised--ec.js

```diff
@@ -0,0 +1,246 @@
+            },
+          ],
+        },
+        {
+          title: 'Follow us on',
+          title_with_separator: true,
+          links: [
+            {
+              link: {
+                label: 'Facebook',
+                path: exampleLink,
+                aria_label: 'Link to Facebook',
+                icon_position: 'before',
+              },
+              icon: {
+                path: '/icon-social-media.svg',
+                name: 'facebook-negative',
+                size: 'xs',
+              },
+            },
+            {
+              link: {
+                label: 'Twitter',
+                path: exampleLink,
+                aria_label: 'Link to Twitter',
+                icon_position: 'before',
+              },
+              icon: {
+                path: '/icon-social-media.svg',
+                name: 'twitter-negative',
+                size: 'xs',
+              },
+            },
+            {
+              link: {
+                label: 'Linkedin',
+                path: exampleLink,
+                aria_label: 'Link to Linkedin',
+                icon_position: 'before',
+              },
+              icon: {
+                path: '/icon-social-media.svg',
+                name: 'linkedin-negative',
+                size: 'xs',
+              },
+            },
+          ],
+          links_inline: true,
+        },
+      ],
+      [
+        {
+          title: 'About us',
+          title_with_separator: true,
+          links: [
+            {
+              link: {
+                label: 'Information about the DG',
+                path: exampleLink,
+                aria_label: 'Link to Information about the DG',
+              },
+            },
+          ],
+        },
+        {
+          title: 'Related sites',
+          title_with_separator: true,
+          links: [
+            {
+              link: {
+                label: 'Related link 1',
+                path: exampleLink,
+                aria_label: 'Link to Related link 1',
+              },
+            },
+            {
+              link: {
+                label: 'Related link 2',
+                path: exampleLink,
+                aria_label: 'Link to Related link 2',
+              },
+            },
+            {
+              link: {
+                label: 'Related link 3',
+                path: exampleLink,
+                aria_label: 'Link to Related link 3',
+              },
+            },
+            {
+              link: {
+                label: 'Related link 4',
+                path: exampleLink,
+                aria_label: 'Link to Related link 4',
+              },
+            },
+            {
+              link: {
+                label: 'Related link 5',
+                path: exampleLink,
+                aria_label: 'Link to Related link 5',
+              },
+            },
+          ],
+        },
+      ],
+    ],
+    [
+      [
+        {
+          links: [
```

Comment:
I didn't check the specs, but I guess that this section is optional (there is no control to disable it)

Reply:
for the actual standardised version, this section is mandatory, so I guess we should keep it like this. https://ec.europa.eu/component-library/ec/standardised-template/footer/usage/

Reply:
for the harmonised I don't think that it is mandatory. To be confirmed with designers I guess

Reply:
new control `show_class_names` added